src/shmem_internal.h

```diff
@@ -170,44 +170,38 @@ extern unsigned int shmem_internal_rand_seed;
 #define SHMEM_ERR_CHECK_INITIALIZED()                                   \
     do {                                                                \
         if (!shmem_internal_initialized) {                              \
-            fprintf(stderr, "ERROR: %s(): " PACKAGE_NAME " library not initialized\n", \
-                    __func__);                                          \
-            abort();                                                    \
+            RAISE_ERROR_STR(PACKAGE_NAME " library not initialized\n"); \
```

Comment:
I think this is the only place where we cant reuse these macros. The SHMEM runtime has not been initialized, so we cant call `shmem_runtime_abort`. May be we can do
```
RETURN_ERROR_STR
abort()
```